build/commands/lib/util.js

```diff
@@ -28,6 +28,36 @@ const mergeWithDefault = (options) => {
   return Object.assign({}, config.defaultOptions, options)
 }
 
+const downloadWebDiscoveryExtension = () => {
+  const projectName = 'web-discovery-project'
+  const webDiscoveryGitUrl = 'git@github.com:brave/humanweb.git'
+  const packageJson = path.join(config.braveCoreDir, 'package.json')
+  const version = util.getPackageJsonDependencyVersion(packageJson, projectName)
+  if (!version) {
+    Log.error('Could not find web-discovery-project version')
+    process.exit(1)
+  }
+  const gitTag = 'v' + version
+
+  const braveExtensionPath = 'components/brave_extension/extension/brave_extension'
+  const braveExtensionDir = path.join(config.braveCoreDir, braveExtensionPath)
+  const webDiscoveryDir = path.join(braveExtensionDir, projectName)
+  if (fs.existsSync(webDiscoveryDir)) {
+    Log.progress(`Removing existing project at ${webDiscoveryDir}`)
+    fs.rmdirSync(webDiscoveryDir, { recursive: true })
+  }
+  Log.progress(`Cloning ${projectName} version ${version} to ${braveExtensionDir}`)
+  util.run('git', [
+    '-C', braveExtensionDir,
+    'clone',
+    '-b', gitTag, '--single-branch',
+    webDiscoveryGitUrl, projectName
+  ])
+  Log.progress(`Building ${projectName}`)
+  util.runNpm(webDiscoveryDir, ['install'])
+  util.runNpm(webDiscoveryDir, ['run', 'build-module'])
+}
```

Comment:
I am still unsure as to why this is needed. Wouldn't it work to use npm to manage the dependency directly? i.e. `npm install --save github:brave/humanweb#v1.1.0` (which will clone the repo behind the scene at the specified tag and populate the `node_modules` folder alongside other dependencies).

This works for me in a clean package but maybe I'm missing some Brave-specific requirements/constraints.